Hints.py

```diff
@@ -559,6 +600,26 @@ def buildWorldGossipHints(spoiler, world, checkedLocations=None):
 
     random.shuffle(stoneIDs)
 
+    #Create list of items for which we want hints. If Bingosync URL is supplied, include items specific to that bingo. If not (or if the URL is invalid), use generic bingo hints
+    if world.hint_dist=="bingo":
+        bingoDefaults=read_json("data/Bingo/generic_bingo_hints.json")
```

Comment:
Use `data_path` to retrieve the root data/ path.

Reply:
Fixed